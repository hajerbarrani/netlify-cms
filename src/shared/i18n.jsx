/* eslint-disable */
import i18n from 'i18next'
import Backend from 'i18next-xhr-backend'
import { reactI18nextModule } from 'react-i18next'
import LanguageDetector from 'i18next-browser-languagedetector'
import fr from './locales/fr.json'
import en from './locales/en.json'

i18n
  .use(Backend)
  .use(LanguageDetector)
  .use(reactI18nextModule)
  .init({
    fallbackLng: "fr",
    resources: {
      fr: {
        translations: fr
      },
      en: {
        translations: en
      }
    },
    ns: ["translations"],
    defaultNS: "translations",
    debug: false,
    interpolation: {
      escapeValue: false,
    },
    react: {
      wait: true,
    }
  })
export default i18n
