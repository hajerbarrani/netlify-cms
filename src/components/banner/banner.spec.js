import React from 'react'
import renderer from 'react-test-renderer'
import { shallow, mount, render } from 'enzyme'
import HomeBanner from './banner'
import { I18nextProvider } from 'react-i18next'
import i18n from '../../shared/i18n'


describe('HomeBanner', () =>

  it('should renders correctly', () => {
    const tree = renderer.create(
      <I18nextProvider i18n={ i18n }>
        <HomeBanner />
      </I18nextProvider>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

)
