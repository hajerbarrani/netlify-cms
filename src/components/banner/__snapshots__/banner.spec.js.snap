// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeBanner should renders correctly 1`] = `
<div
  className="banner"
>
  <h1
    className="h1"
  >
    Faire des travaux n'a jamais été aussi simple
  </h1>
  <h2
    className="h2"
  >
    Artisans qualifiés, architectes, accompagnement de A à Z, services à la carte... Homly You s'adapte à tous vos besoins.
  </h2>
  <button
    className="cta"
    type="button"
  >
    Nos services
  </button>
</div>
`;
